FEM/rf_random_walk.cpp

```diff
@@ -5980,8 +6026,9 @@ void DATWriteParticleFile(int current_time_step)
 	sprintf(now, "%i", current_time_step);
 	string nowstr = now;
 
-	string vtk_file_name =  pathJoin(defaultOutputPath, FileName);
-	vtk_file_name += "_RWPT_" + nowstr + ".particles.vtk";
+	string vtk_file_name = FileName + "_RWPT_";
```

Comment:
These changes cause the failing of the RWPT benchmarks. With the changes, the path of the output of particle solution is set to where the input files are in the ctests, and the particle solution files are written to the corresponding directories of benchmarks not to the building directory.   Recovering to the original definition of `vtk_file_name` (lines 5983-5984 ) makes the RWPT benchmarks work.